polaris-react/src/components/BulkActions/BulkActions.scss

```diff
@@ -7,76 +7,55 @@ $bulk-actions-button-stacking-order: (
 
 .Group {
   @include text-style-input;
-  width: 100%;
-  display: none;
+  display: flex;
   align-items: center;
   flex-wrap: wrap;
   opacity: 0;
+  width: 100%;
+  justify-content: center;
+  transition: var(--p-duration-250) var(--p-ease-in);
+  transition-property: transform, opacity;
```

Comment:
We've seen some issues recently on safari, so we'll want to make sure to thoroughly tophat. `will-change` is something we may want to take advantage of, but we should keep an eye on CPU usage. This is only one element, so I don't suspect it'll to high, however, we've had perf issues in the past with react transition group - so let's keep our eyes open!